src/sidebar/components/HelpPanel.js

```diff
@@ -112,41 +128,33 @@ function HelpPanel({ auth, session }) {
       panelName="help"
       onActiveChanged={onActiveChanged}
     >
-      <div className="hyp-u-vertical-spacing HelpPanel__content">
-        <div className="hyp-u-layout-row--align-center">
-          <h3 className="HelpPanel__sub-panel-title">
+      <div className="space-y-4">
```

Comment:
I hope a `Block` component will be forthcoming that will encapsulate this spacing, which we use in various places.